src/c_kzg_4844.c

```diff
@@ -590,12 +590,13 @@ static uint32_t reverse_bits(uint32_t n) {
  * @param[in,out] values The array, which is re-ordered in-place
  * @param[in]     size   The size in bytes of an element of the array
  * @param[in]     n      The length of the array, must be a power of two
- *                       less that 2^32
+ *                       less that 2^32 and unequal to 1.
  */
 static C_KZG_RET bit_reversal_permutation(
     void *values, size_t size, uint64_t n
 ) {
     CHECK(n >> 32 == 0);
+    CHECK(n != 0);
```

Comment:
Nit, could we swap the order of these? I prefer checks be in order of fast/simple -> slow/complex.

```suggestion
    CHECK(n != 0);
    CHECK(n >> 32 == 0);
```